Makefile

```diff
@@ -11,11 +11,18 @@ ifneq ($(RELEASE_VERSION),latest)
 endif
 
 .PHONY: all
-all: java_verify docker_build
+all: java_verify copy_crd docker_build docker_tag docker_push
+
+.PHONY: build
+build: java_verify copy_crd docker_build
 
 .PHONY: clean
 clean: java_clean
 
+.PHONY: copy_crd
+copy_crd:
```

Comment:
We have a similar step in operators called `crd_install`. Would it make sense to keep the naming?

Long term, it would be also good to have a check in the pipeline that the files are up-to-date. Not needed in this PR, but maybe opning an issue to not forget abotu it would be nice.

Reply:
Yes I will align the naming and open an issue.